Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,65 @@
+#include <iostream>
+#include "Person.h"
+
+using namespace std;
+
+Person::Person(const std::string& name, int age)
+{
+    age_ = new int(2);
+    *age_ = age;
```

Comment:
You can replace these two lines with just: `age_ = new int(age);` to initialize age_ to a pointer pointing to age.